samples/photo_album/src/components/PhotosUploader.js

```diff
@@ -82,7 +82,7 @@ class PhotosUploader extends Component {
                     {this.props.uploadedPhotos.map(uploadedPhoto => {
                         return (
                             <UploadedPhotoStatusContainer
-                                key={uploadedPhoto.public_id}
+                                key={uploadedPhoto.fileName}
```

Comment:
@felixmosh
File name might not be unique,  can you please change key to be the index of map?

Reply:
To use index as react's key is considered as a bad practice, but for demo sake it enough

Reply:
@felixmosh merged. thanks for the pr and changes!